.travis.yml

```diff
@@ -6,4 +6,28 @@ rvm:
   - 2.0.0
   - 2.1.6
   - 2.2.2
-
+  - jruby-19mode
+  - jruby-9.0.0.0
+  - rbx-2
+jdk:
+  - openjdk7
```

Comment:
there are no java specific features / feels like a it of overkill ... how about latest jdk only ?